fictadvisor-api/src/v2/utils/documentation/permission/check.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiDocumentationParams } from '../decorators';
```

Comment:
Name of the documentation file (check.ts) must be the same as corresponding controller method (checkPermissions.ts)